app/views/plugins/petitions/_petition.liquid

```diff
@@ -47,6 +47,7 @@
           akid:             window.sumofus.personalization.urlParams.akid,
           source:           window.sumofus.personalization.urlParams.source,
           thermometer:      window.sumofus.personalization.thermometer,
+          cosmetic:         true,
```

Comment:
Oh wow, I actually read that line and still asked. Derp. OK, I have no comments on this, but your specs are failing on CircleCI.